webrender/res/prim_shared.glsl

```diff
@@ -727,11 +727,12 @@ TransformVertexInfo write_transform_vertex(RectWithSize instance_rect,
 
 struct ResourceRect {
     vec4 uv_rect;
+    vec2 user_data;
```

Comment:
I agree with the sentiment, but since glyph handling is so high in the profile (it's typically 60+% of the frame time), I think making this path as fast as possible is probably worth it here. Perhaps I could rename it to be glyph offset specifically, since I don't see it ever being used for something else?